djangocms_alias/filters.py

```diff
@@ -76,3 +80,36 @@ def choices(self, changelist):
                 ),
                 "display": title,
             }
+
+
+if djangocms_versioning_enabled:
+    from djangocms_versioning.constants import UNPUBLISHED
+
+    class UnpublishedFilter(admin.SimpleListFilter):
+        title = _("unpublished")
+        parameter_name = "unpublished"
+
+        def lookups(self, request, model_admin):
+            return (("1", _("Show")),)
```

Comment:
I'm not so sure about using 1, a reusable / future proofed alternative could have been to make this the version filter that only allows filtering by unpublished. No point wasting time changing this now, if more options are required in the future that person / request can rewrite the filter.